src/components/tabs/_tabs.scss

```diff
@@ -72,7 +72,7 @@
     list-style: none;
     display: flex;
     flex-direction: column;
-
+    z-index: 1;
```

Comment:
Can you just add a line break after line 75 😄

Reply:
done!

Reply:
Should we add a `z-index` number in to https://github.com/carbon-design-system/carbon-components/blob/e5dca27/src/globals/scss/_layout.scss#L54-L63 and put a reference to that in here...?

Reply:
Nice, I haven't seen that z-index variable. 
Maybe I should replace `z-index: 1` with `z-index: z('dropdown'); `because that is basically a dropdown on small screens...

Reply:
I think it makes sense - Please don't forget testing the change with wider-screen, too. Thanks!

Reply:
Yeah, it has a problem on wide screens with the `z-index: z('dropdown');` 
If you add tabs after a list-box the tabs will be rendered above the drop-down items because both elements have the same z-index and the last item in the DOM, shows on top.

Will add the the `z-index` to `auto` on the `bp--sm--major` breakpoint.